Extensions/xkit_patches.js

```diff
@@ -161,17 +161,10 @@ XKit.extensions.xkit_patches = new Object({
 		}, 1000);
 	},
 
-	run_order: ["7.8.1", "7.8.2", "7.9.0"],
+	run_order: ["7.8.1", "7.8.2", "7.9.1"],
```

Comment:
7.9.0 needs to be in there so that it doesn't run 7.8.x patches

Reply:
i guess this means we need an empty 7.9.0 function too. sorry that my patches structure isnt nice for this type of core patch :sweat: